### DISCUSSION TITLE
Power Entitiy from Tongou 2P 63A Circuit Breaker DIN Relay?

### DISCUSSION THREAD

**Discussion by jay7210** [arendst/Tasmota#20747] [Q&A] [answered]
Ive flashed Tasmota to this and used the  config from https://templates.blakadder.com/tongou_TO-Q-SR263JWE.html. Is there any way to get the power reading from the RCD as its not showing up as a entity in HA. Switch (ON/OFF) is working fine.

Cheers

**Comment by barbudor** [comment]
If your device matches the one of the template, there is no power measurement
Should there be, you would see it on the main page of the tasmota main page

**Comment by jay7210** [comment]
The Circuit breaker in question is an Ewelink one that does have an energy meter inside it, but the config has not assigned a GPIO to it and i do not know what to assign to activate it.

**Reply by barbudor** [reply]
Can you point to the right page then
Because the device pointed by your link does not have an energy meter

**Reply by barbudor** [reply]
OK, so the product description seems to say that it has power measurement
Which means that the tasmota template is incomplete

As you have already opened the device to flash tasmota, you need to identify which power measurment chip is used and how it is connected to the ESP

**Reply by sfromis** [reply]
The Aliexpress sales page linked from that template says:
> **Notice:**
> This type have no energy meter function
> Can not set current ,voltage ,power in app

Of course, it may be a different device....

**Reply by barbudor** [reply]
I see multiple listing's for this reference with

![Screenshot_2024-02-17-20-56-36-740_com alibaba aliexpresshd-edit](https://github.com/arendst/Tasmota/assets/2996042/f86ff093-3e23-4bc0-b45d-569e98d8ae3e)

So only an analysis of the hardware can answer

**Comment by jay7210** [comment, marked as answer]
Sorted it;

{"NAME":"RCD_CONTACTOR","GPIO":[32,224,0,3104,0,0,0,0,288,544,0,0,0,0],"FLAG":0,"BASE":18}

for anyone who wants the basic Tongou 2P 63A Circuit Breaker DIN Relay and power monitoring.

**Reply by barbudor** [reply]
Sent a PR to upddate the template: https://github.com/blakadder/templates/pull/1850
